### PR TITLE
Week 4 retrospective update with Paul's changes

### PR DESCRIPTION
**Pull request by joeycor92**
Hi Folks, 

Week 4 retrospective as per the discussion on Friday.

As discussed in the Discord chat, I've put the 'Start' section under actions but understand that a few of these don't belong.

Discussion to be had here regarding changes to be made.

Thanks
Joey

### CODE REVIEW

File: sprintRetrospective/week4.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+- GitHub (everyone):
+  - Delete Branches when done
+  - Pull requests often, and for small components
+- Readme files for subfolders. Include anything needed to run locally (everyone)
+- Let people know that there's only a small review needed. (everyone)
+- Include as many images as possible, e.g. console with no errors, terminal with no errors (everyone)
+- Prototype - Get a user evaluation before starting the page (everyone)
+- Finish off full pages. Don't leave things things unfinished for weeks(everyone)
+- Communicate with the team as to where the data will be used. Who will need the data and in what format will it be needed. Check before you start doing it (everyone)
+- Create spreadsheet of routes for both teams (WHO?)
```
Comment (BearachB) [inline sprintRetrospective/week4.md:61]: For this point, I think @joeycor92 you said you'd make it this morning in the stand-up? Or did I misunderstand?

**Comment by WarrenK-design** [comment]
image
Took car of this, excel can be found [here](https://tudublin-my.sharepoint.com/:x:/r/personal/c16463344_mytudublin_ie/_layouts/15/Doc.aspx?sourcedoc=%7Ba378ec28-ca20-4e78-ad21-3dd6dd132b0f%7D&action=editnew&cid=1c75c861-4a25-4667-abdc-04581eef02e8) so we can use going forward when route complete. 

The fields are as shown below. 
image

**Review by BearachB — APPROVED** [review]
Good work Joey! 
Not sure if we'd need to address this, but is it possible we're putting too much detail into the retro? I know for myself, I'm going to struggle to keep all the starts/stops/actions on the top of my head without referring back to the document constantly.

**Review by WarrenK-design — APPROVED** [review]
Looks good

**Comment by joeycor92** [comment]
> Good work Joey! Not sure if we'd need to address this, but is it possible we're putting too much detail into the retro? I know for myself, I'm going to struggle to keep all the starts/stops/actions on the top of my head without referring back to the document constantly.

I think it's definitely a discussion for Friday. It's too big at the moment. But also hopefully the starts and stops will get smaller, and the continue sections will get bigger but they will be things we're already doing and won't need to look back on them. They'll just come natural to us.

**Review by DaraghK93 — APPROVED** [review]
looks great Joey, very thorough. I think you're right, if we remember to put these into action this week, it won't be as long ext week

**Pull request merged**